lib/shopify_api/message_enricher.rb

```diff
@@ -1,11 +1,17 @@
 module ShopifyAPI
   class MessageEnricher < SimpleDelegator
     def message
-      return super unless (400...500).include?(code.to_i)
+      return super if code.to_i != 422
```

Comment:
I would prefer to keep this within the whole range, 422 may be the only one we support currently by the docs but that doesnt mean we won't be adding it in the future.